### DISCUSSION TITLE
Animate placeholder transition

### DISCUSSION THREAD

**Discussion by stanislavlysenko0912** [NuPlay/RichText#26] [Q&A] [answered]
How to animate placeholder change?
When html loaded, than frames height change instantly, and placeholder hide instantly too.
I want too add frame height animation, and fade to hide placeholder animation.

Can i do this? If can, how 🙂? 

<img src="https://user-images.githubusercontent.com/86171768/216831701-6bd6df41-90a9-4a07-a03e-f2b395a5d2f8.gif" width="230" height="100">

**Comment by NuPlay** [comment, marked as answer]
I added transition modifier in version 2.1.2 #27
You can use it like this 😆

```swift
import SwiftUI
import RichText

struct RichText_Test: View {
    @State private var  html = ""
    
    var body: some View {
       ScrollView{
            RichText(html: html)
                .placeholder {
                    Text("loading")
                }
                .transition(.easeOut)
        }
    }
}

struct RichText_Test_Previews: PreviewProvider {
    static var previews: some View {
        RichText_Test()
    }
}

```
